lookup_plugins/aws_sg_id.py

```diff
@@ -33,18 +35,19 @@ def get_group_id(self, group_name):
         # TODO error checking
         conn = boto.ec2.connect_to_region(self.region)
 
-        sg = conn.get_all_security_groups(groupnames=group_name)[0]
+        sgs = conn.get_all_security_groups()
+        filtered = filter(lambda x: x.name==group_name, sgs)[0]
```

Comment:
Is there a reason to do this client side rather than (I think the filter does it) serverside?